usecase/test/test_work_session.py

```diff
@@ -31,6 +31,15 @@ def setUp(self):
         usecase.reset()
         self.usecase = usecase
 
+    def test_usecase_is_usecase(self):
```

Comment:
Nice, method name...

Reply:
If you like it so much, you could add that one to your own as well ;)

Reply:
I guess, but shouldn't you better call it `test_usecase_is_instance`?